source/fundamentals/indexes.txt

```diff
@@ -371,3 +384,53 @@ The following example creates a unique, descending index on the ``theaterId`` fi
 
 Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB server manual for more information.
 .. driver-content-end
+
+Remove an Index
+---------------
+
+Pass the indexed ``name`` field to the ``dropIndex()`` method to remove
+an index from a collection. 
+
+Use the ``listIndexes()`` method to see the indexed ``name`` fields in
+your collection. 
+
+The following snippet retrieves all the indexes in your collection:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin listIndex
+   :end-before: end listIndex
+
+Your output should look something like the following:
```

Comment:
Since they're not specifying a sample collection that has a text index on it, I'm not sure we can claim what the index text looks like. Maybe you can set up this example by providing some background information about the collection that you're calling listIndexes on.